doc/apiv3-documentation.apib

```diff
@@ -548,6 +548,7 @@ a client should be able to discover further resources in the API.
 | Link                | Description                                      | Type          | Nullable | Supported operations |
 |:-------------------:| ------------------------------------------------ | ------------- | -------- | -------------------- |
 | configuration       | The configuration of this OpenProject instance   | Configuration |          | READ                 |
+| my_preferences      | The preferences of the logged-in user            | MyPreferences |          | READ                 |
```

Comment:
Will we hide this link for the anonymous user? I guess so... In other tables we used a *condition* column to indicate when you can't see a certain link.

The alternative (I thought about) would be to nulify the link for the anonymous user. I am undecided here. For properties that do not exist we usually `null` them. For actions that can't be performed (or properties that can't be seen for privacy/permission reasons) we hide the link.

I'd go for hiding I guess...